asserts/snap_asserts.go

```diff
@@ -99,9 +99,101 @@ func buildSnapBuild(assert assertionBase) (Assertion, error) {
 	}, nil
 }
 
+// SnapRevision holds a snap-revision assertion, asserting the properties of a
+// snap release by a store authority.
+type SnapRevision struct {
+	assertionBase
+	sequence  uint64
+	timestamp time.Time
+}
+
+// SnapID returns the snap id of the built snap.
+func (assert *SnapRevision) SnapID() string {
+	return assert.Header("snap-id")
+}
+
+// SnapDigest returns the digest of the built snap.
+func (assert *SnapRevision) SnapDigest() string {
+	return assert.Header("snap-digest")
+}
+
+// Sequence returns the sequence number.
+func (assert *SnapRevision) Sequence() uint64 {
+	return assert.sequence
+}
+
+// SnapBuild returns the digest of the associated snap-build.
+func (assert *SnapRevision) SnapBuild() string {
+	return assert.Header("snap-build")
+}
+
+// DeveloperID returns the developer's ID.
```

Comment:
// DeveloperID returns the id of the developer that submitted the snap build to the store.

(is that right?)